scripts/check_ecosystem.py

```diff
@@ -121,17 +121,23 @@ async def _get_commit(self: Self, checkout_dir: Path) -> str:
     Repository("aiven", "aiven-client", "main"),
     Repository("alteryx", "featuretools", "main"),
     Repository("apache", "airflow", "main", select="ALL"),
+    Repository("apache", "airflow", "superset", select="ALL"),
```

Comment:
I think you meant to use `apache/superset` if I'm not mistaken.
```suggestion
    Repository("apache", "superset", "master", select="ALL"),
```